diff --git a/default/applications/mimeapps.list b/default/applications/mimeapps.list
index 61079e8..3c3945e 100644
--- a/default/applications/mimeapps.list
+++ b/default/applications/mimeapps.list
@@ -12,7 +12,6 @@ application/pdf=org.gnome.Evince.desktop
 
 x-scheme-handler/http=chromium.desktop
 x-scheme-handler/https=chromium.desktop
-x-scheme-handler/mailto=HEY.desktop
 
 video/mp4=mpv.desktop
 video/x-msvideo=mpv.desktop
